integration-test.sh

```diff
@@ -3,9 +3,13 @@ set -e
 
 DIR=$PWD
 
-curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash
-cd ./fabric-samples/first-network
-echo y | ./byfn.sh up
-cp -r crypto-config "$DIR" && cd "$DIR"
+curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash -s -- 2.1.1 1.4.7 0.4.20
```

Comment:
why do need to specify version here (instead of default value in `bootstrap.sh`?

Reply:
will try